plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/Completions.hs

```diff
@@ -207,7 +209,7 @@ splitAtPosition pos ls = do
   split <- splitFile
   pure $ reverse $ Rope.lines $ fst split
   where
-    splitFile = Rope.splitAtPosition ropePos ls
+    splitFile = Rope.utf16SplitAtPosition ropePos ls
```

Comment:
I think I set this up right, but I'm not 100% sure. We would need a cabal file test with unicode characters...